template.json

```diff
@@ -1,15 +1,21 @@
 {
-  "dependencies": {
-    "@craco/craco": "^5.6.4",
-    "autoprefixer": "^9.8.6",
-    "tailwindcss": "^1.7.3",
-    "@testing-library/react": "^10.4.9",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/user-event": "^12.1.3"
-  },
-  "scripts": {
-    "start": "craco start",
-    "build": "craco build",
-    "test": "craco test"
+  "package": {
+    "dependencies": {
+      "@craco/craco": "^5.6.4",
+      "autoprefixer": "^9.8.6",
+      "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.0.1",
+      "@testing-library/react": "^11.2.2",
+      "@testing-library/jest-dom": "^5.11.4",
+      "@testing-library/user-event": "^12.1.3",
+      "web-vitals": "^1.0.1"
+    },
+    "scripts": {
+      "start": "craco start",
+      "build": "craco build",
+      "test": "craco test"
+    },
+    "eslintConfig": {
```

Comment:
This is added as per the default [cra-template](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template.json)